services/cooperation.js

```diff
@@ -0,0 +1,51 @@
+const Cooperation = require('~/models/cooperation')
+const { createError } = require('~/utils/errorsHelper')
+const { COOPERATION_NOT_FOUND } = require('~/consts/errors')
+
+const cooperationService = {
+  getCooperations: async () => {
+    const cooperations = await Cooperation.find().lean().exec()
+
+    return cooperations
+  },
+
+  getCooperationById: async (id) => {
+    const cooperation = await Cooperation.findById(id).lean().exec()
+
+    if (!cooperation) {
+      throw createError(404, COOPERATION_NOT_FOUND)
+    }
+
+    return cooperation
+  },
+
+  createCooperation: async (offerId, tutorId, studentId, price, status) => {
+    const newCooperation = await Cooperation.create({
+      offerId,
+      tutorId,
+      studentId,
+      price,
+      status
+    })
+
+    return newCooperation
+  },
+
+  updateCooperation: async (id, updateData) => {
```

Comment:
gotta use filter here for the 'updateData', that only price and status can change